styles.css

```diff
@@ -0,0 +1,298 @@
+    Geneva,
+    Verdana,
+    sans-serif;
+  font-style: normal;
+  line-height: 50px;
+}
+
+.ellipse_3 p {
+  background-image: url(./image/Ellipse\ 12.png);
+  background-repeat: no-repeat;
+  background-position: 100%;
+  background-position-y: bottom;
+  background-position-x: right;
+  font-family:
+    "Lucida Sans",
+    "Lucida Sans Regular",
+    "Lucida Grande",
+    "Lucida Sans Unicode",
+    Geneva,
+    Verdana,
+    sans-serif;
+  width: 75%;
+  height: 145px;
+  font-size: 16px;
+  padding-left: 24px;
+  padding-right: 24px;
+  padding-bottom: 30px;
+}
+
+.ellipse {
+  background-image: url(./image/Rectangle\ 7.png);
+  background-repeat: no-repeat;
+  background-position: 100%;
+  background-position-y: calc(10%);
+  background-position-x: center;
+}
+
+.ellipse_zero {
+  padding-left: 200px;
+  position: calc(50%);
+}
+
+.ellipse_zero_1 {
+  padding-left: 150px;
+  position: calc(50%);
+  margin-top: 50px;
+}
+
+.ellipse_1 {
+  background-image: url(./image/Ellipse\ 11.png);
+  background-size: 10%;
+  background-repeat: no-repeat;
+  background-position-y: calc(10%);
+}
+
+.ellipse_2 {
+  background-image: url(./image/Ellipse\ 4.png);
+  background-repeat: no-repeat;
+  background-position: 100%;
+  background-position-y: calc(85%);
+  background-position-x: center;
+}
+
+.ellipse_3 {
+  background-image: url(./image/Ellipse\ 3.png);
+  background-repeat: no-repeat;
+  background-position: 100%;
+  background-position-y: calc(-1%);
+}
+
+.ellipse,
+.ellipse_3,
+.ellipse_2,
+.ellipse_1 {
+  width: 100%;
+  height: 718px;
+}
+
+.contact_wraper {
+  background-image: url(./image/Rectangle\ 5.png);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-position-y: bottom;
+  height: 190px;
+}
+
+.contact {
+  padding-left: 10px;
+  background-image: url(./image/Ellipse\ 8.png);
+  background-repeat: no-repeat;
+  background-position-y: center;
+  background-position-x: calc(10%);
+}
+
+ul {
+  padding: 10px;
+  margin: 0;
+}
+
+.connect {
+  padding-left: 13px;
+}
+
+.social_media {
+  padding-top: 10px;
+  list-style: none;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 150px;
+}
+
+.social_media_list a img {
+  padding: 20px, 20px, 20px 0;
+  width: 25px;
+  height: 20px;
+}
+
+.vector {
+  width: 14px;
+  height: 20px;
+}
+
+/* work section start  */
+
+/* .work img {
+  width: 100%;
+  padding-top: 10px;
+} */
+
+.work h1 {
+  color: rgb(41, 41, 106);
+  font-weight: bold;
+}
+
+.profile {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 300px;
+  list-style: none;
+}
+
+.skill {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 250px;
+  text-decoration: none;
+  list-style: none;
+}
+
+/* .btn {
+  height: 30px;
+  font-size: 15px;
+}
+
+.btn a {
+  text-decoration: none;
+  color: rgb(104, 143, 215);
+  font-weight: 600;
+} */
+
+/* .btn a:hover {
+  color: rgb(208, 139, 36);
+} */
```

Comment:
- [ ] [OPTIONAL] Kindly remove all commented code from your codebase to make it neater 🙏